# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1481907867 18000
#      Fri Dec 16 12:04:27 2016 -0500
# Node ID 1632fb20b60b403ed34ba600aa08b2866a9b1c22
# Parent  7766e3ef6c69c605323bba899a11aaf92860ec17
# Parent  ecc19f427bd670b70d56a34fbbf63f9b7cef87f5
maint: Periodic merge of stable to default.

diff --git a/libgui/languages/de_DE.ts b/libgui/languages/de_DE.ts
--- a/libgui/languages/de_DE.ts
+++ b/libgui/languages/de_DE.ts
@@ -2791,17 +2791,17 @@ oder wenn unterschiedliche Schriftgrößen oder -stile (fett/kursiv/normal) verwendet werden.</translation>
     <message>
         <location filename="../src/settings-dialog.cc" line="+127"/>
         <source>Octave Settings</source>
         <translation>Octave Einstellungen</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>Unable to save settings.  Missing settings file or unknown directory.</source>
-        <translation>Die Einstellungen können nicht gespeichert werden.  Fehlende Datei oder unbekanntes Verzeichnis.</translation>
+        <translation>Die Einstellungen können nicht gespeichert werden. Fehlende Datei oder unbekanntes Verzeichnis.</translation>
     </message>
     <message>
         <location line="+18"/>
         <location line="+4"/>
         <location line="+444"/>
         <source>System setting</source>
         <translation>Systemeinstellung</translation>
     </message>
@@ -2877,17 +2877,17 @@ a:link { text-decoration: underline; col
 &lt;head/&gt;&lt;body&gt;
 &lt;p&gt;Allow Octave to connect to the Octave web site when it starts to display current news and information about the Octave community.&lt;/p&gt;
 &lt;/body&gt;&lt;/html&gt;</source>
         <translation>&lt;html&gt;&lt;head&gt;
 &lt;style&gt;
 a:link { text-decoration: underline; color: #0000ff; }
 &lt;/style&gt;
 &lt;head/&gt;&lt;body&gt;
-&lt;p&gt;Erlaube Octave beim Programmstart eine Verbindung zur Octave Webseite herzustellen um aktuelle Neuigkeiten und Informationen über Octave anzuzeigen.&lt;/p&gt;
+&lt;p&gt;Erlaube Octave beim Programmstart eine Verbindung zur Octave Webseite herzustellen, um aktuelle Neuigkeiten und Informationen über Octave anzuzeigen.&lt;/p&gt;
 &lt;/body&gt;&lt;/html&gt;</translation>
     </message>
 </context>
 <context>
     <name>shortcut_manager</name>
     <message>
         <location filename="../src/shortcut-manager.cc" line="+105"/>
         <source>New File</source>
diff --git a/liboctave/module.mk b/liboctave/module.mk
--- a/liboctave/module.mk
+++ b/liboctave/module.mk
@@ -78,17 +78,17 @@ liboctave_liboctave_la_LIBADD += \
   libgnu/libgnu.la \
   $(LIBOCTAVE_LINK_DEPS)
 
 # Increment these as needed and according to the rules in the libtool manual:
 liboctave_liboctave_current = 4
 liboctave_liboctave_revision = 0
 liboctave_liboctave_age = 0
 
-liboctave_liboctave_version_info = $(liboctave_liboctave_current):$(liboctave_liboctave_revision):$(liboctave_age)
+liboctave_liboctave_version_info = $(liboctave_liboctave_current):$(liboctave_liboctave_revision):$(liboctave_liboctave_age)
 
 liboctave_liboctave_la_LDFLAGS = \
   -version-info $(liboctave_liboctave_version_info) \
   $(NO_UNDEFINED_LDFLAG) \
   @XTRA_CRUFT_SH_LDFLAGS@ \
   -bindir $(bindir) \
   $(LIBOCTAVE_LINK_OPTS) \
   $(WARN_LDFLAGS)
diff --git a/scripts/ode/private/integrate_adaptive.m b/scripts/ode/private/integrate_adaptive.m
--- a/scripts/ode/private/integrate_adaptive.m
+++ b/scripts/ode/private/integrate_adaptive.m
@@ -97,17 +97,17 @@ function solution = integrate_adaptive (
       solution.retout = x;
     endif
     feval (options.OutputFcn, tspan, solution.retout, "init",
            options.funarguments{:});
   endif
 
   ## Initialize the EventFcn
   if (! isempty (options.Events))
-    ode_event_handler (options.Events, tspan(end), x,
+    ode_event_handler (options.Events, tspan(1), x,
                        "init", options.funarguments{:});
   endif
 
   if (options.havenonnegative)
     nn = options.NonNegative;
   endif
 
   solution.cntloop = 0;
